src/components/dataset/query/sidebar/QueryViewSidebarItem.jsx

```diff
@@ -17,15 +27,28 @@ export class QueryViewSidebarItem extends React.PureComponent {
     token: PropTypes.string,
   };
 
-  handleChange = value => {
-    const { column, handleChange, tableName } = this.props;
+  componentDidUpdate(prevProps, prevState) {
```

Comment:
Can you add some comments in the code on why these functions are necessary? At a high level I understand that by default the reset button is disabled, but when a filter is applied, this updates the filterMap which in turn updates the state of the reset button to enabled. Is this correct?

Reply:
Yes, that's right! However, at the design meeting yesterday we decided to omit the reset button, so I will update this and add some comments.